### ISSUE TITLE
Documentation Colour Contrast

### ISSUE THREAD

**Issue by danielleplatt95** [laravel/ideas#1814]
To make the documentation more accessible to people who have sight-related disabilities, or people who have sight issues, the contrast between the background on the segments of code and the background on the page should be higher. As it stands, this currently has a [contrast ratio of 1.03:1](https://webaim.org/resources/contrastchecker/?fcolor=FBFBFD&bcolor=FFFFFF). 

The contrast between the colour of the font for code comments and the colour of the code background has a [contrast of 2.94:1](https://webaim.org/resources/contrastchecker/?fcolor=93939E&bcolor=FBFBFD). This contrast should also be made greater to make it more accessible.

**Comment by mfn** [comment]
And interesting tidbit is that, since the redesign, the background-color is not set. I'm using a custom one (this green-ish think you see) and, except the red links :), it's quite pleasant to read:
![image](https://user-images.githubusercontent.com/87493/64241815-b7ae8f00-cf04-11e9-94ee-4a37bbaba5be.png)

**Comment by erdemkeren** [comment]
That would make my eyes happy.

**Comment by svenluijten** [comment]
The site is open source, I'm sure Taylor would accept a PR with this info in it as well: https://github.com/laravel/laravel.com-next
